docs/pages/server-access/guides/jetbrains-sftp.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: JetBrains SFTP
+description: How to use JetBrains IDE to browse remote host with Teleport
+h1: SFTP with JetBrains IDE
+---
+
+This guide explains how to use Teleport and JetBrains IDE to access files with SFTP.
```

Comment:
For users who discover this guide after already visiting the docs site, I think we should add another sentence to the first paragraph about why using Teleport with SFTP and a JetBrains IDE is useful.